SHARDING.md

```diff
@@ -0,0 +1,254 @@
+# Sharding Playbook
+
+This playbook walks through the steps to create a new log shard.
+The steps should be the same for Rekor and the CT log, though the
+CT process for generating key material may differ.
```

Comment:
rekor shouldn't know about CT, or this should be in another repo?

Reply:
Will update, just wanted to state that the Rekor and CT sharding processes should be the same.